server/models/workers.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require('mongoose');
+
+// Schema for workers
+const WorkerSchema = new mongoose.Schema({
+    Fname: {
+        type: String,
+        required: true
+    },
+    Lname: {
+        type: String,
+        required: true
+    },
+    phone: {
+        type: Number,
+        required: true,
+        unique: true,
+        min: 10,
+        max: 10
+    },
+    email: {
+        type: String,
+        unique: true,
+        lowercase: true,
+        required: true
+    },
+    location: {
+        type: Array,
+        required: true
+    },
+    timeSlot: {
+        type: Array,
+        required: true
+    },
+    subService: {
+        type: Array,
+        required: true
+    },
+    Available: {
```

Comment:
Can we make `Available` attribute as an array so that we can use time slot there

Reply:
`Available` attribute is used to check if the worker is currently assigned work or not. if it is true he is not assigned any work and he is available else he is currently assigned a work. `timeSlot` attribute is already provided for storing the available time slots in which the worker is available